jenkins-builds/production.sh

```diff
@@ -7,18 +7,19 @@ scriptname=$(basename $0)
 
 usage() {
     echo "Usage: $0 [OPTIONS] <list-of-ebfiles>
-    -a,--arch     Architecture (gpu or mc)      (mandatory: Dom and Piz Daint only)
-    -f,--force    Force build of given package  (optional: double quotes for a list)
+    -a,--arch     Architecture (gpu or mc)           (mandatory: Dom and Piz Daint only)
+    -f,--force    Force build of given package       (optional: double quotes for a list)
     -h,--help     Help message
-    -l,--list     Production list file          (mandatory: EasyBuild production list)
-    -p,--prefix   EasyBuild prefix folder       (mandatory: installation folder)
-    -x,--xalt.    [yes|no] update XALT database (optional, default is yes)
+    -l,--list     Production list file               (mandatory: EasyBuild production list)
+    -p,--prefix   EasyBuild prefix folder            (mandatory: installation folder)
+    -u,--unuse    Module unuse colon separated PATH  (optional: default is null)
```

Comment:
Is there a case where -u and -x will not be used together? 

If not, then I'd prefer to have only one option to that replaces both options (such as `-r,--regression`).

But I'm ok with this solution as well.